flixel/input/FlxPointer.hx

```diff
@@ -15,6 +15,7 @@ class FlxPointer
 	
 	private var _globalScreenX:Int = 0;
 	private var _globalScreenY:Int = 0;
+	private static var _cachedPoint:FlxPoint = new FlxPoint();
```

Comment:
I'd rather stick with the ```_cached``` convention, if there are a ```_point``` variables that are static for caching purposes, we should rename it to ```_cachedPoint```.